surveys/views.py

```diff
@@ -138,6 +138,19 @@ def get_success_url(self):
         return str(success_url)
 
 
+class SurveyPropertiesEdit(UpdateView):
+    model = SurveyFormEntry
+    template_name = "survey_properties_edit.html"
+    form_class = SurveyCreateForm
+    context_object_name = 'form_object'
+
+    def get_success_url(self):
+        form_entry_id = self.object.pk
+        success_url = reverse_lazy('fobi.edit_form_entry', kwargs={'form_entry_id': form_entry_id})
+
+        return str(success_url)
```

Comment:
What's the reason why the URL needs to be cast to `str`?

Reply:
It's unnecessary, good catch! I think just copied over from an example without checking.